src/agentscope/message/msg.py

```diff
@@ -228,6 +228,18 @@ def formatted_str(self, colored: bool = False) -> str:
                 colored_strs.append(f"{name}: {self.url}")
         return "\n".join(colored_strs)
 
+    def __eq__(self, value: object) -> bool:
```

Comment:
What if additional attributes are added to the message in the future?

Reply:
If there are new attributes, this method must be modified at the same time